### PR TITLE
Fix intersection observer margin to absolute for non-document-root supporting platforms

### PR DESCRIPTION
**Pull request by dvoytenko**
Fixes https://github.com/ampproject/error-reporting/issues/20

### CODE REVIEW

File: src/service/loading-indicator.js [hunk 1]
```diff
@@ -68,12 +67,9 @@ export class LoadingIndicatorImpl {
...
     let io;
     try {
-      io = new win.IntersectionObserver(ioCallback, {
-        root,
-        rootMargin,
-      });
+      io = new win.IntersectionObserver(ioCallback, {root, rootMargin: '25%'});
     } catch (e) {
-      io = new win.IntersectionObserver(ioCallback, {rootMargin});
+      io = new win.IntersectionObserver(ioCallback, {rootMargin: '150px'});
```
Comment (jridgewell) [inline src/service/loading-indicator.js:72]: Nit: Can we use `25vw 25vh`?

**Review by jridgewell — APPROVED** [review]

File: src/service/loading-indicator.js (hunk 1 above)
Comment (samouri) [reply src/service/loading-indicator.js:72]: why would this be allowed but `25%` be banned?

**Review by samouri — APPROVED** [review]

File: src/service/loading-indicator.js (hunk 1 above)
Comment (dvoytenko) [reply src/service/loading-indicator.js:72]: Trying to confirm what we can and cannot use as values for those browsers. So far no luck reproducing it even on Chromium  50.0.2661.0. It's possible that this is an old Android WebView that's WebKit based.

Comment (dvoytenko) [reply src/service/loading-indicator.js:72]: Keeping `px` for max performance. `150px` is a good default around 25%.

**Pull request merged**